### PR TITLE
Rename screenshot files

### LINKED ISSUE #118: Change default name of screenshot
Might be good to update the default name of the screenshot, for example <config_name>.png, or perhaps also date stamp?

### PR DESCRIPTION
**Pull request by DanSava**
Closes #118

### CODE REVIEW

File: everviz/plugins/plugin_abc.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    """
+
+    def __init__(self):
+        super().__init__()
+        self._screenshot_filename = f"{self.plugin_name()}.png"
+        ASSETS_DIR = Path(pkg_resources.resource_filename("everviz", "assets"))
+        WEBVIZ_ASSETS.add(ASSETS_DIR / "axis_customization.css")
+
+    def plugin_name(self):
+        name = map(lambda c: f"_{c.lower()}" if c.isupper() else c, type(self).__name__)
```
Comment (verveerpj) [inline everviz/plugins/plugin_abc.py:28]: why the map? is `__name__` not just a string, i.e. `name = __name__.lower()` does not work?

**Review by verveerpj — COMMENTED** [review]
I did not get this to work, even after installing the latest webviz, is the version with the necessary changes not released yet? Apart from that, the code seems fine, one small comment.

File: everviz/plugins/plugin_abc.py (hunk 1 above)
Comment (DanSava) [reply everviz/plugins/plugin_abc.py:28]: it does work but I want to replace `SummaryPlot` with `_summary_plot` and not `summaryplot`

**Comment by DanSava** [comment]
> I did not get this to work, even after installing the latest webviz, is the version with the necessary changes not released yet? Apart from that, the code seems fine, one small comment.

This will only work using the master branch from webviz-config and also the latest webviz-core-config. 

For webviz-config the changes are merged into master but no new version was uploaded to pypip so they will not be there if you just pip install it (or at least that was true last week did not check it this morning)

File: everviz/plugins/plugin_abc.py (hunk 1 above)
Comment (verveerpj) [reply everviz/plugins/plugin_abc.py:28]: ah, now I see what you are doing...

**Review by verveerpj — APPROVED** [review]

**Pull request merged**